### ISSUE TITLE
Improving visualization on data preparation step

### ISSUE THREAD

**Issue by CKPEIIKA** [quantum-entangled/machine-learning-ui#46]
Line plots can be sometimes not so informative and we may make an option to show a scatter plot instead or just make connected scatterplot. We also can show a histogram when x and y axis are the same variable. There are other types of helpful visualizations (e.g. heatmap, boxplots), but they probably will require a separated widget.

**Comment by quantum-entangled** [comment]
Yes, it's definitely a feature to implement, line plots were just migrated from the previous version. But I would suggest not to separate widgets, but either use `Stack` (you can examine it in `ManageLayersWidget`, `SelectOptimizerWidget`) and add a distinct dropdown for plot type or just pass the plot type variable to plotting function/s.
